Formula/c/clickhouse-sql-parser.rb

```diff
@@ -0,0 +1,19 @@
+class ClickhouseSqlParser < Formula
+  desc "Writing clickhouse sql parser in pure Go"
+  homepage "https://github.com/AfterShip/clickhouse-sql-parser"
+  url "https://github.com/AfterShip/clickhouse-sql-parser/archive/refs/tags/v0.3.8.tar.gz"
```

Comment:
@git-hulk can you mark 0.3.8 as latest release?

Reply:
@chenrui333 Sure, it's done now. Thank you!